packages/grant-explorer/src/features/discovery/LandingPage.tsx

```diff
@@ -1,165 +1,101 @@
-import Footer from "common/src/components/Footer";
-import { lazy, Suspense, useState, useEffect } from "react";
+import { useEffect } from "react";
 import {
-  getActiveRounds,
-  getRoundsInApplicationPhase,
-  RoundOverview,
+  useActiveRounds,
+  usePrefetchRoundsMetadata,
+  // useRoundsEndingSoon,
+  useRoundsTakingApplications,
 } from "../api/rounds";
-import { useDebugMode } from "../api/utils";
-import Navbar from "../common/Navbar";
-import ActiveRoundsSection from "./ActiveRoundSection";
-import ApplyNowSection from "./ApplyNowSection";
-import { ROUND_PAYOUT_DIRECT, ROUND_PAYOUT_MERKLE } from "common";
-import useSWR, { mutate } from "swr";
-const LandingBannerLogo = lazy(() => import("../../assets/LandingBanner"));
-
-export function useActiveRounds() {
-  const debugModeEnabled = useDebugMode();
-  return useSWR(`activeRounds`, () => getActiveRounds(debugModeEnabled));
-}
-
-export function useRoundsInApplicationPhase() {
-  const debugModeEnabled = useDebugMode();
-  return useSWR(`applicationRounds`, () =>
-    getRoundsInApplicationPhase(debugModeEnabled)
-  );
-}
+import { DefaultLayout } from "../common/DefaultLayout";
+import LandingHero from "./LandingHero";
+import { LandingSection, ViewAllLink } from "./LandingSection";
+import RoundCard from "./RoundCard";
+import { useLocation } from "react-router-dom";
 
 const LandingPage = () => {
+  const location = useLocation();
   useEffect(() => {
-    if (process.env.REACT_APP_ENV === "production") {
+    if (
+      process.env.REACT_APP_ENV === "production" &&
+      !location.search.includes("skip_redirect")
+    ) {
       window.location.replace("https://grants.gitcoin.co");
     }
-  }, []);
-
-  const [searchQuery, setSearchQuery] = useState("");
+  }, [location]);
 
-  const { isLoading: activeRoundsLoading, data: activeRounds } =
-    useActiveRounds();
+  usePrefetchRoundsMetadata();
```

Comment:
This will start loading all the metadata in the background.

Before the metadata for each round was fetched all at once. This could mean 50+ requests that had to be completed before showing these in the UI.

Now we can get the metadata from the RoundCard when it's being displayed in the view.